docs/resources/object_bucket_policy.md

```diff
@@ -0,0 +1,103 @@
+---
+page_title: "Scaleway: scaleway_object_bucket_policy"
+description: |-
+Manages Scaleway object storage bucket policy.
+---
+
+# scaleway_object_bucket
+
+Creates and manages Scaleway object storage bucket policy.
+For more information, see [the documentation](https://www.scaleway.com/en/docs/storage/object/api-cli/using-bucket-policies/).
+
+## Example Usage
+
+```hcl
+resource "scaleway_object_bucket_policy" "bucket" {
+  name = "some-unique-name"
+}
+
+resource "scaleway_object_bucket_policy" "bucket" {
```

Comment:
Is it supposed to be the resource type and identifier than the previous one ?

Reply:
not really important for the hcl config but wrong for the example. Thanks

Reply:
I've just renamed